packages/aws-client/tests/test-services.js

```diff
@@ -188,6 +188,7 @@ test('ec2() service defaults to localstack in test mode', async (t) => {
   );
 });
 
+// can this test be removed or is it needed for the granule CloudMetrics queries?
```

Comment:
With es instance going away, I think packages/aws-client/tests/services.js es and this test are not needed.

Reply:
when I tried removing ES, I found that some of the ES vars were for Metrics, so it seemed like ES needs to stay for these tests

Reply:
ES references removed